canonical-websites/jaas.ai
# Update site navigation

## Done

- Added global nav as an ES6 import
- Change primary nav bg color
- Change logo
- Rework primary nav items to match updated designs

## QA

- Run site with `./run`
- View site and verify global and primary nav appears as:

<img width="1266" alt="screenshot 2019-03-06 at 13 22 57" src="https://user-images.githubusercontent.com/505570/53884257-186f1480-4013-11e9-9efc-31d777a0897f.png">

Fixes https://github.com/ubuntudesign/juju-squad/issues/521
Fixes https://github.com/ubuntudesign/juju-squad/issues/519
